app/scripts/components/logs/viewer.js

```diff
@@ -141,7 +141,11 @@ var LogViewer = React.createClass({
             if (text.length > 200) {
               text = text.slice(0, 200) + '...';
             }
-            const level = d.level ? d.level.toLowerCase() : 'info';
+            let level;
+            if (d.level) {
+              if (d.level === 30) level = 'info';
+              else level = d.level.toString().toLowerCase();
```

Comment:
I think this still fails if the level is integer but has a value other than 30.

Reply:
It won't fail, It will just output the number as a string instead of "error" or whatever else. 
`40 2018-05-21T19:27:31.417Z	ed63b375-e577-4a65-8142-dd8c25b5b6b8	{"level":"info","timestamp":"2018-05-21T19:27:31.407Z","sender":"undefined","message":"Received the provider { provider: 's3_provider' }"}`